README.rst

```diff
@@ -30,6 +30,31 @@ Included components
 * `qemu-lite`_: Includes an optimized version of the QEMU hypervisor.
 
 
+Repositories
+============
+
+* Ubuntu
+
+  * 16.04 http://download.opensuse.org/repositories/home:/clearcontainers:/clear-containers-3/xUbuntu_16.04
+  * 16.10 http://download.opensuse.org/repositories/home:/clearcontainers:/clear-containers-3/xUbuntu_16.10 
+  * 17.04 http://download.opensuse.org/repositories/home:/clearcontainers:/clear-containers-3/xUbuntu_17.04
+
+* Fedora
+
+  * 24 http://download.opensuse.org/repositories/home:/clearcontainers:/clear-containers-3/Fedora_24
+  * 25 http://download.opensuse.org/repositories/home:/clearcontainers:/clear-containers-3/Fedora_25
+  * 26 http://download.opensuse.org/repositories/home:/clearcontainers:/clear-containers-3/Fedora_26
+
+* CentOS
+
+  * 7.x http://download.opensuse.org/repositories/home:/clearcontainers:/clear-containers-3/CentOS_7/
+
+* RHEL
+
+  * 7.x http://download.opensuse.org/repositories/home:/clearcontainers:/clear-containers-3/RHEL_7/
+
+* OpenSUSE
```

Comment:
yes is still missing we do not have packages built yet in the "non-staging" repo.